src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1454,14 +1450,14 @@ private static Binder GetQueryEnclosingBinder(int position, CSharpSyntaxNode sta
                                                   position, unbound.BindForErrorRecovery().Binder, unbound.Syntax);
                     case BoundKind.Lambda:
                         var lambda = (BoundLambda)n;
-                        return GetEnclosingBinder(AdjustStartingNodeAccordingToNewRoot(startingNode, lambda.Body.Syntax), 
+                        return GetEnclosingBinder(AdjustStartingNodeAccordingToNewRoot(startingNode, lambda.Body.Syntax),
                                                   position, lambda.Binder, lambda.Body.Syntax);
                     default:
                         goto done;
                 }
             }
 
-done:
+            done:
```

Comment:
I think the code is more readable when labels are at the beginning of the lines.